apps/explorer/components/ExplorerDatum.tsx

```diff
@@ -48,13 +50,16 @@ export function ExplorerDatum({
             <ValueCopyable
               scaleSize="18"
               label={entityType}
-              href={getHref(entityType, entityValue)}
-              value={entityValue}
-              displayValue={
-                entityType === 'block' && entityValue
-                  ? Number(entityValue).toLocaleString()
-                  : entityValue
+              href={
```

Comment:
Got this covered. One thought for the future: it may be preferable from a UX perspective to rewrite the URL on Block to either always use the block or always use the blockHash as the id. Could also redirect, but that's a bit heavier than needed, maybe. I'd personally want the functionality to be such that if you clicked the link for the "other" route going to this same page, nothing would happen. Would need to look into the right strategy for that. It's a bit of a feeling, but it just seems like a different ID on the same base route like this should go somewhere else or it just shouldn't be a visible thing. Internally, the functionality of accepting both is excellent.

Reply:
Yeah true, thats a change probably worth exploring.